Sources/NIO/Thread.swift

```diff
@@ -130,25 +130,33 @@ final class Thread {
 /// returned by `currentValue` is defined per thread.
 ///
 /// - note: `ThreadSpecificVariable` has reference semantics.
```

Comment:
@weissi you can remove this one now as its a class and so this is true always ;)

Reply:
done